Documentation/design-docs/native-hosting.md

```diff
@@ -306,6 +307,38 @@ Closes a host context.
 * `host_context_handle` - handle to the initialized host context to close.
 
 
+### Loading managed components
+To load managed components from native app directly (not using COM or WinRT) the hosting layer exposes a new runtime helper/delegate `hdt_load_assembly_and_get_function_pointer`. Calling the `hostfxr_get_runtime_delegate(handle, hdt_load_assembly_and_get_function_pointer, &helper)` returns a function pointer to the runtime helper with this signature:
+```C
+int load_assembly_and_get_function_pointer_fn(
```

Comment:
Do we want to talk about x86 calling convention?

Reply:
I'm intentionally leaving out the technical details - I think it doesn't belong to this document. Ideally we should have a header file which defines that in programmatic way which anybody can understand perfectly.